lte/cloud/go/services/subscriberdb/streamer/providers.go

```diff
@@ -133,8 +133,8 @@ func subscriberToMconfig(ent configurator.NetworkEntity, apnConfigs map[string]*
 		AuthOpc:  cfg.Lte.AuthOpc,
 	}
 
-	if cfg.Lte.SubProfile != "" {
-		sub.SubProfile = string(cfg.Lte.SubProfile)
+	if cfg.Lte.SubProfile != nil && *cfg.Lte.SubProfile != "" {
```

Comment:
Yeah the fact that all these get introduced as ptrs now is def annoying... no input from Googling/searching through GH Issues?

Reply:
Good question. We're currently not sure exactly what lead to this happening, so it requires some more digging. See also https://github.com/magma/magma/pull/12151#discussion_r850172178